src/tools/pkcs11-tool.c

```diff
@@ -96,9 +96,15 @@ extern CK_FUNCTION_LIST_3_0 pkcs11_function_list_3_0;
 #define MAX_TEST_THREADS 10
 #endif
 
+#ifndef MIN
+# define MIN(a, b)	(((a) < (b))? (a) : (b))
+#endif
```

Comment:
We have this defined three times throught the codebase already:
```
src/libopensc/internal.h:#define MIN(x, y) (((x) < (y)) ? (x) : (y))
src/pkcs15init/pkcs15-cardos.c:# define MIN(a, b)       (((a) < (b))? (a) : (b))
src/pkcs15init/pkcs15-incrypto34.c:# define MIN(a, b)   (((a) < (b))? (a) : (b))
```
could the `internal.h` implementation be reused? Many other tools and places already include the `internal.h`.